### DISCUSSION TITLE
Avaliação em pares

### DISCUSSION THREAD

**Discussion by jesielviana** [ifpi-picos/ads-programacao-para-internet-i#33] [Show and tell]
Gostaria de solicitar que cada aluno avaliasse seu próprio projeto e pelo menos um trabalho de outro colega e retornasse um feedback sobre as avaliações para o colega avaliado e para mim (professor) e se sentir confortável pode compartilhar o feedback aqui.

**Pontos para considerar na avaliação:**

- Layout/Design (combinação de cores, organização dos elementos, etc)
- Responsividade
- Conteúdo
- Código (nomes, organização, boas práticas, etc)
- Relevância do trabalho

**Comment by Rafhael0069** [comment]
Rafhael Gaspar, avaliando projeto de Rodrigo Pereira: https://github.com/RodrigoPereira25/rodrigopereira25.github.io
O design base é bem estabelecido e conhecido, mas ainda está em desenvolvimento, por isso ainda não é muito bom visualmente.
Já tem responsividade e responde bem, assim como o tópico acima ainda precisa de ajustes.
O conteúdo é simples mas bem exposto sobre a área.
O código é bem legível e de fácil entendimento, as classes e ids são intuitivas e o código é bem comentado.
O projeto é bem relevante pois mostra o domínio sobre o assunto e mostra o progresso feito.

**Comment by ghost** [comment]
Avaliação de Portfólio
Avaliador: Rodrigo Pereira
Avaliado: Rafhael Gaspar │ Link do repositório: https://github.com/Rafhael0069/portfolio.git

Layout/Design: o layout utilizado foi bem definido e bem pensado, considerando a interação do usuário com a página.
Responsividade: a página é responsiva, com características bem estabelecidas e funciona muito bem em qualquer tamanho de tela.
Conteúdo: o conteúdo é bem simples, porém é bem organizado e exposto.
Código: o código está bem organizado, mas seria interessante deixar um pouco mais comentado.
Relevância do trabalho: a relevância do projeto reflete tanto o aprendizado do aluno quanto no preparo para o mercado de trabalho!

**Reply by jesielviana** [reply]
ótimo, Rodrigo. parabéns pela dedicação e trabalho realizado.

**Comment by WesleyVeloso30** [comment]
Avaliação de Portfólio
: Wesley
Avaliado: Iago Brendo
│ Link de avaliação: https://iagobrenoad.github.io

O layout ficou bem organizado, estruturado, resultando em um fácil entendimento e manuseio. 

O código ficou bem distribuído com a utilização certa das tags.

O conteúdo é bem simples, porém é bem organizado e exposto.

A responsabilidade do atual site não está muito boa,  pois não se adequa às menores telas de celular, porém com a utilização de "@media" no CSS e posteriormente um pouco de Javascript o site ficará perfeitamente responsivo.

**Reply by jesielviana** [reply]
@iagobrenoad precisa ajustar a responsividade do menu e o contraste da cor da fonte e background o menu.

**Reply by IagoBrenoAD** [reply]
Ok, com o tempo eu vou ajustando ele

**Comment by Allan-Barbosa** [comment]
Avaliação de Portfólio
Avaliador: Allan Barbosa
Avaliado: Alessandro Silva
Site: https://alessandrovianasilva.github.io/
Layout: O layout ficou bem organizado e com uma boa combinação de cores.
Responsividade: O site foi feito utilizando porcentagens como medida, porém a falta de media queries fez com que o site não fosse totalmente responsivo.
Conteúdo: O site tem pouco conteúdo, mas acredito ser o essencial.
Código: O código está bem organizado e com nomes de classes que representam bem sua função.
Relevância do trabalho: Neste trabalho o aluno demonstrou estar aprendendo o conteúdo ensinado em aula, então o trabalho foi relevante.

**Reply by jesielviana** [reply]
@alessandrovianasilva está faltando ajustar a responsividade do menu

**Reply by AlessandroVianaSilva** [reply]
@jesielviana Estarei deixando completo em relação à responsividade nas próximas alterações.

**Comment by AlessandroVianaSilva** [comment]
Auto-Avaliação do projeto https://alessandrovianasilva.github.io/
Layout: Layout organizado e com cores que combinam.
Responsividade: O site não está totalmente responsivo, mas nas próximas alterações estará completo em relação a isso.
Conteúdo: O site é simples, porém essencial com os tópicos Inicio, Sobre Mim, Experiência e Contatos, com seus respectivos conteúdos,, um abaixo do outro.
Código: Código bem organizado com os nomes facilmente de identificar, em ordem de acordo tanto no index.HTML E CSS. 
Relevância do trabalho: Neste trabalho, estou demonstrando o que estou aprendendo tanto em sala de aula quanto fora dela, o que faz parte de um desenvolvedor web.




Avaliação do projeto: https://github.com/Allan-Barbosa/Allan-Barbosa.github.io
Layout: layout simples, mas bastante conteúdo em uma tela só; o que poderia ser feito: Colocar os tópicos um abaixo do outro para melhor destaque de cada, e uma melhor distribuição dos tópicos na navbar, sendo que falta para poder coloca-lo na navbar.
Conteúdo: Conteudo está bastante diversificado, apesar da navbar, estar somente os conteúdos de: Sobre, Projetos e Contatos, deixando de lado os outros tópicos existentes no site/portfolio.
Responsividade: Em relação a isso, está responsivo, apesar nao ter usado código do javascript.
Código: Código bem organizado, com classes fácil de entender.
Relevância do trabalho: Está demonstrando o que está sendo ensinado em sala de aula, trabalho muito relevante.

**Comment by IagoBrenoAD** [comment]
Avaliação de Portfólio
Avaliador: Iago Breno
Avaliado: Wesley Veloso | Link do repositório: https://github.com/WesleyVeloso30/wesleyveloso30.github.io
Layout/Design- Possui um Layout simples, objetivo e de fácil entendimento para o usuário, porém com melhorias a se considerar na parte visual, como por exemplo a escolha e qualidade da imagem.
Responsividade- Possui uma boa responsividade, com características um pouco mais a frente da linha de estudos até então estabelecida (o uso do JavaScript), se adequando muito bem às telas.
Conteúdo- Seu conteúdo é básico, mas claro, muito bem organizado e que vai direto ao ponto.
Código- Ficou muito bem estruturado e nítido, cumprindo assim com as boas práticas.
Relevância do trabalho- O trabalho é relevante pois mostra claramente o avanço do discente em relação aos assuntos abordados.

**Comment by marcielysilva** [comment]
Avaliação do portefólio: Ezequias
Avaliador: Marciely Ferreira
link da avaliação: https://ezequiasmoura.github.io/

Layout/Design: A combinação de elementos e design está bom, deveria colocar o nome em destaque(ficar mais visível);
Responsabilidade: Mostra bem e responde bem os comando que tem e a tela projecta bem para todos os tamanhos de tela;
Conteúdo: O conteúdo é bom, direto e simples;
Código: é de fácil compreensão e comentado os códigos.
Relevância do trabalho: é bom e mostra domínio no que fez durante o processo.

**Comment by ezequiasmoura** [comment]
Avaliação do portifólio: Marciely
Avaliador: Ezequias S. Moura 
Link do portifólio avaliado: https://marcielysilva.github.io/

Layout/Desing: As cores estão bem atrativas para a finalidade do seu projeto e o layout de compreensão fácil porém precisa de ajustes na parte da largura da página.
Responsividade: o site se mostra responsivo na parte do menu no entanto na parte do conteúdo do site precisa melhorar um pouco.
Conteúdo: conteúdo de fácil compreensão
Código: código organizado o que facilita na sua compreensão para uma possível manutenção.
Relevância do trabalho: Percebesse que foi bem absolvido o conteúdo de HTML e CSS.
